index.bs

```diff
@@ -1225,16 +1241,20 @@ To verify that an |environment| <dfn>is allowed to show a file picker</dfn>, run
     the selected files.
 
     Additional options can be passed to {{showOpenFilePicker()}} to indicate the types of files
-    the website wants the user to select. See [[#api-filpickeroptions-types]] for details.
+    the website wants the user to select and the preferred directory which the
```

Comment:
Sentence structure is awkward in English. 

How about: "...directory in which the file picker will open."